.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,8 @@
+---
+name: 📚 Documentation
+about: Report an issue related to documentation
+---
+
+## 📚 Documentation
+
```

Comment:
Documentations PRs that have been raised so far, had a ss attached (which was generated  after running it locally), Volunteers made it a standard in GSoD, not sure it should be the standard for future PRs as well

Reply:
This repository is different from https://github.com/wechaty/wechaty.js.org so not always we will be getting screenshots so we need `documentation.md` for gRPC API docs issues and all.